module/zfs/vdev_initialize.c

```diff
@@ -46,7 +46,7 @@ unsigned long zfs_initialize_value = 0xdeadbeefdeadbeeeULL;
 int zfs_initialize_limit = 1;
 
 /* size of initializing writes; default 1MiB, see zfs_remove_max_segment */
-uint64_t zfs_initialize_chunk_size = 1024 * 1024;
+unsigned long zfs_initialize_chunk_size = 1024 * 1024;
```

Comment:
Unrelated to your change, but it seems like this could default to SPA_MAXBLOCKSIZE (on ZoL), like zfs_remove_max_segment.